ios/RNPurchases.m

```diff
@@ -329,17 +293,17 @@ - (dispatch_queue_t)methodQueue {
 
 #pragma mark -
 #pragma mark Delegate Methods
-- (void)purchases:(RCPurchases *)purchases didReceiveUpdatedPurchaserInfo:(RCPurchaserInfo *)purchaserInfo {
-    [self sendEventWithName:RNPurchasesPurchaserInfoUpdatedEvent body:purchaserInfo.dictionary];
+- (void)purchases:(RCPurchases *)purchases didReceiveUpdatedCustomerInfo:(RCCustomerInfo *)customerInfo {
+    [self sendEventWithName:RNPurchasesCustomerInfoUpdatedEvent body:customerInfo.dictionary];
 }
 
-- (void)         purchases:(RCPurchases *)purchases
-shouldPurchasePromoProduct:(SKProduct *)product
-            defermentBlock:(RCDeferredPromotionalPurchaseBlock)makeDeferredPurchase {
+- (void)purchases:(RCPurchases *)purchases
+readyForPromotedProduct:(RCStoreProduct *)product
+         purchase:(void (^)(void (^ _Nonnull)(RCStoreTransaction * _Nullable, RCCustomerInfo * _Nullable, NSError * _Nullable, BOOL)))startPurchase {
```

Comment:
we should update the indentation here for consistency

Reply:
How so? The second parameter doesn't fit. Or do you mean something else?